docs/en/query_language/table_functions/file.md

```diff
@@ -1,7 +1,7 @@
 
 # file
 
-Creates a table from a file.
+Creates a table from a file. This table function is similar to [hdfs](hdfs.md) one.
```

Comment:
```suggestion
Creates a table from a file. This table function is similar to [HDFS](hdfs.md) one.
```

what about url?

Reply:
Not so similar because of globs, but I would add it.

Table function hdfs named hdfs, not HDFS, do we need capital letters here?

Reply:
> Table function hdfs named hdfs, not HDFS, do we need capital letters here?

@stavrolia probably not, but that looks weird and might be confusing, probably worth to make it case-insensitive.